homeassistant/components/cybro/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""Support for Cybro PLC."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import CybroDataUpdateCoordinator
+
+PLATFORMS = (
+    Platform.BINARY_SENSOR,
+    # Platform.BUTTON,
+    Platform.LIGHT,
+    # Platform.NUMBER,
+    # Platform.SELECT,
+    Platform.SENSOR,
+    # Platform.SWITCH,
+    Platform.WEATHER,
+)
+
+PLATFORM_NAMES = ("binary_sensor", "button", "light", "number", "sensor", "switch")
```

Comment:
Can you start with a single platform (e.g. binary_sensor)? Your current PR is to big to review.

Reply:
Yes sure, then i will adapt it to use just the binary_sensor which is creating just some diagnosis data.
Is it ok to exclude the other platforms? Or shall i remove also the source files?

Reply:
Done, now only one platform is left (other files are removed)